src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  margin-top: 20px;
+  position: relative;
+  font-family: Avenir, sans-serif;
+}
+
+.big-bar {
+  height: 70px;
+  width: 100%;
```

Comment:
a lot of properties are repeated in big-bar and small-bar they could be written in another class like "bar" and you can use this class for both bars, that makes your code shorter